app/layout.tsx

```diff
@@ -44,11 +44,7 @@ export default function RootLayout({ children }: RootLayoutProps) {
         >
           <TanstackProviders>
             <ThemeProvider attribute="class" defaultTheme="system" enableSystem>
-              <div className="relative flex h-full flex-col">
-                <SiteHeader />
-
-                <main>{children}</main>
-              </div>
+              {children}
```

Comment:
`main` 태그까지는 괜찮지 않을까요?

```suggestion
              <main>{children}</main>
```

Reply:
기존에 `SiteHeader` 가 `main` 바깥 영역에 위치해있어 동일하게 옮겼습니다!
`RootLayout` 내 `AuthLayout` 이 종속되기 때문에 위의 경우 `SiteHeader` 가 `main` 요소에 포함될 수 있는데 문제 없다면 위와 같이 진행하겠습니다!